AEPEdge.xcodeproj/project.pbxproj

```diff
@@ -1571,7 +1571,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0.0.beta.2;
+				MARKETING_VERSION = 1.0.0.3;
```

Comment:
I kept the build version, @shalehaha do you plan to do the same for core extensions?